torch/nn/quantized/dynamic/modules/rnn.py

```diff
@@ -267,15 +267,15 @@ def from_float(cls, mod):
             mod, 'qconfig'), 'Input float module must have qconfig defined'
 
         if mod.qconfig is not None and mod.qconfig.weight is not None:
-            weight_observer = mod.qconfig.weight()
+            weight_observer_method = mod.qconfig.weight
         else:
             # We have the circular import issues if we import the qconfig in the beginning of this file:
             # https://github.com/pytorch/pytorch/pull/24231. The current workaround is to postpone the
             # import until we need it.
             from torch.quantization.qconfig import default_dynamic_qconfig
-            weight_observer = default_dynamic_qconfig.weight()
+            weight_observer_method = default_dynamic_qconfig.weight
 
-        dtype = weight_observer.dtype
+        dtype = weight_observer_method().dtype
```

Comment:
is there a guarantee that the `weight` is callable? The QConfig is a NamedTuple, so it will take whatever weight you give it. But that doesn't mean it can be called, right?

Reply:
Oh nevermind -- ignore this comment